src/DefaultBuilder/src/WebApplicationBuilder.cs

```diff
@@ -49,6 +49,9 @@ internal WebApplicationBuilder(Assembly? callingAssembly, string[]? args = null)
 
             // Add default services
             _deferredHostBuilder.ConfigureDefaults(args);
+
+            // Enable changes here because we need to pick up configuration sources added by the generic web host
+            _deferredHostBuilder.ConfigurationEnabled = true;
```

Comment:
Do we need this flag at all any more?

Reply:
Yes, we don't want to re-add the default configuration providers again in ConfigureDefaults.

Reply:
And we're okay just assuming ConfigureWebHostDefaults won't do something similar to ConfigureDefaults and add a provider? Doesn't this end up adding two chained configuration sources for `ASPNETCORE_` environment variables.